beacon-chain/rpc/aggregator/server.go

```diff
@@ -76,13 +76,8 @@ func (as *Server) SubmitAggregateAndProof(ctx context.Context, req *pb.Aggregati
 	}
 
 	// Retrieve the unaggregated attestation from pool
-	atts := as.AttPool.UnaggregatedAttestationsBySlotIndex(req.Slot, req.CommitteeIndex)
+	aggregatedAtts := as.AttPool.AggregatedAttestationsBySlotIndex(req.Slot, req.CommitteeIndex)
 
-	// Aggregate the attestations and broadcast them.
-	aggregatedAtts, err := helpers.AggregateAttestations(atts)
```

Comment:
This is the key. Here we don't have to call `helpers.AggregateAttestations` upon every request from aggregators